src/api/telemetry/TelemetryAPI.js

```diff
@@ -337,27 +385,24 @@ export default class TelemetryAPI {
 
         if (!stalenessSubscriber) {
             stalenessSubscriber = this.stalenessSubscriberCache[keyString] = {
-                isStale: false,
                 callbacks: [callback]
             };
             if (provider) {
                 stalenessSubscriber.unsubscribe = provider
-                    .subscribeToStaleness(domainObject, function (isStale) {
-                        stalenessSubscriber.isStale = isStale;
-                        stalenessSubscriber.callbacks.forEach(function (cb) {
+                    .subscribeToStaleness(domainObject, (isStale) => {
```

Comment:
This is no longer just a boolean `isStale`, it's an object with properties

Reply:
Yep! good catch, changing to `stalenessResponse`